docs/releases/1.9.7.txt

```diff
@@ -11,3 +11,6 @@ Bugfixes
 
 * Removed the need for the ``request`` context processor on the admin login
   page to fix a regression in 1.9 (:ticket:`26558`).
+
+* Delayed translation of the password help_text coming from password validators
```

Comment:
Since this applies to AdminPasswordChangeForm or any other forms the user might have, I think you could say:
```
Fixed translation of password validators' ``help_text`` in forms (...).
```